napari/utils/info.py

```diff
@@ -0,0 +1,71 @@
+import platform
+
+import sys
+
+import napari
+
+
+def sys_info(as_html=False):
```

Comment:
Can you add a simple test for this, maybe just that it is callable both with `as_html=True` and without and returns a string in both cases